### DISCUSSION TITLE
Subtitle.

### DISCUSSION THREAD

**Discussion by Badbtc** [Cyfrin/foundry-full-course-cu#15] [Q&A] [answered]
Hello, I'm not sure if this is the right place to ask, but I was wondering when subtitles are added to videos. Thank you!

**Comment by 0xmuneeb** [comment]
I think Patrick's English is itself very clear to understand without subtitles.

**Comment by cromewar** [comment]
Hello @Badbtc YouTube add them automatically, but it might take some time to get processed as they are huge videos, just be a little bit patient.

**Comment by PatrickAlphaC** [comment, marked as answer]
Hello!

If YouTube doesn't auto-generate them, if someone wants to help make them the process looks like this:

1. Dump video into a subtitle generator like [Descript](https://www.descript.com/)
2. Have it output a `.srt` file
3. I'll upload them to YouTube

The issue is Descript gets a _lot_ of stuff wrong so you kind of have to manually go through it too. Might be a huge effort... idk. 


Here is my issue on it:

https://github.com/Cyfrin/foundry-full-course-f23/issues/9

**Comment by Equious** [comment]
Closing discussion since solution was found in linked thread.
